constraints.txt

```diff
@@ -37,7 +37,6 @@ jsonschema==3.2.0
 jstyleson==0.0.2
 kiwisolver==1.3.2
 kwarray==0.5.19
-lxml==4.6.3
```

Comment:
is it worth adding `defusedxml==0.7.1` here?

Reply:
It is already here

Reply:
ok